src/app/routes/getInitialData/index.js

```diff
@@ -18,7 +18,11 @@ const baseUrl = onClient()
   ? getBaseUrl(window.location.origin)
   : process.env.SIMORGH_BASE_URL;
 
-const getUrl = pathname => `${baseUrl}${pathname.replace(ampRegex, '')}.json`;
+const getUrl = pathname => {
+  const url = `${baseUrl}${pathname.replace(ampRegex, '')}.json`;
+  logger.info(`Data: [${url}]`);
```

Comment:
Would it make any difference if we put this after L67? I wouldn't usually expect a function called `getXXX` to  have side-effects.

Reply:
Good shout, I have moved it out of getUrl and into the fetchData function instead